octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -508,10 +507,10 @@ def _convert_book_prices_to_orders(self, book_prices, book_side):
         return [
             {
                 ECOBIC.PRICE.value: float(order_price),
-                ECOBIC.SIZE.value: float(order_size),
+                ECOBIC.SIZE.value: float(book_prices[order_price]),
                 ECOBIC.SIDE.value: book_side,
             }
-            for order_price, order_size in book_prices.items()
+            for order_price in book_prices
```

Comment:
I don't understand why this change ?

Reply:
Good question, I don't remember why, I'll recheck this